Makefile

```diff
@@ -368,7 +368,10 @@ clean-obsolete-protos:
 	@echo "+ $@"
 	$(BASE_DIR)/tools/clean_autogen_protos.py --protos $(BASE_DIR)/proto --generated $(BASE_DIR)/generated
 
-
+.PHONY: update-storage-protolock
+update-storage-protolock: $(PROTOLOCK_BIN)
```

Comment:
should we move this closer to `storage-protos-compatible`? Also, when `storage-protos-compatible` fails, can we add a message saying to run this target instead of directly running `protolock commit`?

Reply:
I waffled on the location.  I'll move it up.  

I will tinker with the error message and see how cumbersome it is to do.